### ISSUE TITLE
How to install? Does this actually move the certs? What about dm-verity?

### ISSUE THREAD

**Issue by T3rm1** [Magisk-Modules-Repo/movecert#8]
Why is there no zip for installing this module?
What about ROMs with dm-verity enabled? Will this module brick the phone if it moves the file to the system folder?

**Comment by techIndia-hacker** [comment]
Yes it actually moves the certificate in /system/etc/security/cacerts/

No OS brick with dmverity on

**Comment by consigliere23** [comment]
If I understood it correctly this should remove user installed certificates to system?
I installed mitmproxy certificate on android emulator and have rooted it.
Can someone help me how to move it now using this tool?

**Comment by techIndia-hacker** [comment]
> 

If u have it rooted with /system partition rewritable u can simply cut paste from user to system certificate store

**Comment by techIndia-hacker** [comment]
> If I understood it correctly this should remove user installed certificates to system? I installed mitmproxy certificate on android emulator and have rooted it. Can someone help me how to move it now using this tool?

If u have it rooted with /system partition rewritable u can simply cut paste from user to system certificate store

Or u can download the src code as zip and flash with magisk

**Comment by consigliere23** [comment]
Yes it's rooted emulator.
I installed this mitmproxy as CA certificate.
Moved from data/misc/user/0/cacerts-added/ to /system/etc/security/cacerts
But still having issue with for example instagram which says "Network error" or similar.
Proxy is working and I set it on WiFi.
Do I need to copy and into this /etc/security/cacerts/ ?
Any recommendation? Thanks :)

**Comment by techIndia-hacker** [comment]
Certain android apps use ssl pinning. That causes custom CA certs untrusted. Eg all FB based apps, bank apps to some extent, other apps may use ssl pinning only for secure domains for payment or user data transfer

**Comment by techIndia-hacker** [comment]
Also chrome since v99 probably doesn't trust custom system certs
